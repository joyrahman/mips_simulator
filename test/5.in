ADDI R1 R0 0
ADDI R2 R0 1023
ADDI R3 R0 1
ADDI R7 R0 2
ADDI R4 R0 0
SUBI R5 R4 32
BEQZ R5 6
AND R6 R2 R3
BEQZ R6 1
ADDI R1 R1 1
MUL R3 R3 R7
ADDI R4 R4 1
BEQZ R0 -8
HLT 
