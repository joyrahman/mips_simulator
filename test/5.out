242
242
572
330
10
1023
0
32
0
0
2
